src/IOHprofiler_csv_logger.h

```diff
@@ -0,0 +1,101 @@
+#ifndef _IOHPROFILER_CSV_LOGGER_H
+#define _IOHPROFILER_CSV_LOGGER_H
+
+#include "IOHprofiler_observer.hpp"
+#include "common.h"
+
+#define BOOST_NO_CXX11_SCOPED_ENUMS
```

Comment:
Please add boost to the requirements in the readme / wiki and include a short instruction of how to install it